fastai/tabular/model.py

```diff
@@ -20,16 +22,30 @@ def _one_emb_sz(classes, n, sz_dict=None):
     return n_cat,sz
 
 # Cell
-def get_emb_sz(to, sz_dict=None):
-    "Get default embedding size from `TabularPreprocessor` `proc` or the ones in `sz_dict`"
+def get_emb_sz(
+    to:(Tabular, TabularPandas), # Tabular or TabularPandas object
+    sz_dict:dict=None # Dict containing embedding sizes to override default emb_sz_rule (i.e. {'catclass1':100})
+) -> list: # List of embedding sizes for each category
+    "Get embedding size for each cat_name in `Tabular` or `TabularPandas`, or populate embedding size manually using sz_dict"
     return [_one_emb_sz(to.classes, n, sz_dict) for n in to.cat_names]
 
 # Cell
 class TabularModel(Module):
     "Basic model for tabular data."
-    def __init__(self, emb_szs, n_cont, out_sz, layers, ps=None, embed_p=0.,
-                 y_range=None, use_bn=True, bn_final=False, bn_cont=True, act_cls=nn.ReLU(inplace=True),
-                 lin_first=True):
+    def __init__(self,
+        emb_szs:list, # Sequence of (num_embeddings, embedding_dim) for each categorical variable
+        n_cont:int, # Number of continuous variables
+        out_sz:int, # Number of outputs for final `LinBnDrop` layer
+        layers:list, # Sequence of ints used to specify the input and output of each `LinBnDrop` layer
```

Comment:
```suggestion
        layers:(int, list), # Used to specify the input size of each `LinBnDrop` layer
```

Reply:
Can layers actually be an int?  I'm not sure the type @muellerzr suggested is accurate.  I actually think I prefer closer to the original 
```suggestion
        layers:list, # Sequence of ints used to specify the input and output size of each `LinBnDrop` layer
```